sdks/python/apache_beam/transforms/external_schematransform_provider_test.py

```diff
@@ -14,24 +14,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 import logging
 import os
+import secrets
+import shutil
+import time
 import unittest
+from importlib import import_module
 
 import pytest
+import yaml
 
 import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions
 from apache_beam.testing.test_pipeline import TestPipeline
 from apache_beam.testing.util import assert_that
 from apache_beam.testing.util import equal_to
 from apache_beam.transforms.external import BeamJarExpansionService
 from apache_beam.transforms.external_schematransform_provider import STANDARD_URN_PATTERN
+from apache_beam.transforms.external_schematransform_provider import ExternalSchemaTransform
 from apache_beam.transforms.external_schematransform_provider import ExternalSchemaTransformProvider
 from apache_beam.transforms.external_schematransform_provider import camel_case_to_snake_case
 from apache_beam.transforms.external_schematransform_provider import infer_name_from_identifier
 from apache_beam.transforms.external_schematransform_provider import snake_case_to_lower_camel_case
 from apache_beam.transforms.external_schematransform_provider import snake_case_to_upper_camel_case
 
+try:
```

Comment:
Why would these not be available?

Reply:
This block imports files that are not in `apache_beam`, which is not supported in some test environments.
From some runs, I see this includes [Python unit tests](https://github.com/apache/beam/blob/445151694fef810d5112ea7e034659f91fceceac/.github/workflows/python_tests.yml#L93) and gradle tasks that run with with [tox environment](https://github.com/apache/beam/blob/445151694fef810d5112ea7e034659f91fceceac/.github/workflows/beam_PreCommit_Python_Transforms.yml#L96) (defined [here](https://github.com/apache/beam/blob/445151694fef810d5112ea7e034659f91fceceac/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L3080))